include/wd_alg_common.h

```diff
@@ -2,6 +2,7 @@
 #define WD_ALG_COMMON_H
 
 #include <asm/types.h>
+#include <pthread.h>
```

Comment:
Why do you need to add pthread head file at here? I can't find the dependency.

Reply:
This demand is required by Wangzhou